troposphere/rds.py

```diff
@@ -167,20 +174,27 @@ class DBInstance(AWSObject):
         'DBSubnetGroupName': (basestring, False),
         'Domain': (basestring, False),
         'DomainIAMRoleName': (basestring, False),
+        'EnableCloudwatchLogExports': (list, False),
```

Comment:
Just like you did for `DBCluster` I think this line should be:
```suggestion
        'EnableCloudwatchLogExports': ([basestring], False),
```

Reply:
done, thanks